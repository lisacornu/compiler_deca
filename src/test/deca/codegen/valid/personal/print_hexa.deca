// Description:
//    Ecriture en utilisant printx et printlnxx
//
// Resultats:
//    Bonjour
//    12
//    2.3
//    1 test 3.3
//
// Historique:
//    cree le 06/01/2025

{
    printx("String : ");
    printlnx("Bonjour");

    printx("Entier : ");
    printlnx(12);

    printx("Float : ");
    printlnx(2.3);

    printx("Plusieurs : ");
    printlnx(1, " test ", 3.3);
}