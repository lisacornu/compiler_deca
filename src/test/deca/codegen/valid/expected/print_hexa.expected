String : Bonjour
Entier : 12
Float : 40133333
Plusieurs : 1 test 40533333
